src/triage/component/catwalk/utils.py

```diff
@@ -202,3 +203,37 @@ def save_db_objects(db_engine, db_objects):
             )
         f.seek(0)
         postgres_copy.copy_from(f, type(db_objects[0]), db_engine, format="csv")
+
+
+def class_path_implements_parallelization(class_path):
+    """Return whether or not the given class_path implements parallelization
+
+    Simply checks whether the path looks like 'RandomForest' or 'ExtraTrees'.
+    The alternative, checking for the existing of 'n_jobs' as a parameter, does not work
+    because it flags LogisticRegression as being parallelizable, but this is only true for
+    multiclass problems.
+
+    Args:
+        class_path (string) a path to a classifier
+
+    Returns: (bool) whether or not the classifier returns parallelization
+    """
+    if 'RandomForest' in class_path or 'ExtraTrees' in class_path:
```

Comment:
🤷‍♂️ or just:

```suggestion
    return 'RandomForest' in class_path or 'ExtraTrees' in class_path
```

Reply:
I would refrain from hardcding the large ones. We could check if the user passed a 'n_jobs' argument in the configuration parameters for the target class_path. This way this functionlity would be equally compatible with xgboost or lightgbm cause they use the 'n_jobs' parameter too.